### ISSUE TITLE
No annotations for detected object 

### ISSUE THREAD

**Issue by xcccc402** [BIOP/qupath-extension-cellpose#13]
Hi: I am trying to use cellpose extension in qupath. It seems like the extension works well and I trained my own model. When I tried to use the model for cell detection, however, no cells are detected (or detected but not annotated) on my images. 

Below is the log:
INFO: Provided model C:\ABBA\QuPath-0.3.2\My\location\cellpose_residual_on_style_on_concatenation_off_train_2022_08_07_23_40_19.334298 is a file. Assuming custom model
INFO: If tiling is necessary, 0 pixels overlap will be taken between tiles
INFO: Folder creation of C:\HistologyData\1085426\cellposetraining\cellpose-temp was interrupted. Either the folder exists or there was a problem.
INFO: Saving images for 6 tiles
INFO: Saving to C:\HistologyData\1085426\cellposetraining\cellpose-temp\Temp_17744_10696.tif
INFO: Saving to C:\HistologyData\1085426\cellposetraining\cellpose-temp\Temp_16954_9672.tif
INFO: Saving to C:\HistologyData\1085426\cellposetraining\cellpose-temp\Temp_17744_8964.tif
INFO: Saving to C:\HistologyData\1085426\cellposetraining\cellpose-temp\Temp_16954_8964.tif
INFO: Saving to C:\HistologyData\1085426\cellposetraining\cellpose-temp\Temp_16954_10696.tif
INFO: Saving to C:\HistologyData\1085426\cellposetraining\cellpose-temp\Temp_17744_9672.tif
INFO: Executing command: [cmd.exe /C C:\Users\xxx\anaconda3\envs\cellpose\python.exe -W ignore -m cellpose --dir C:\HistologyData\1085426\cellposetraining\cellpose-temp --pretrained_model C:/ABBA/QuPath-0.3.2/My/location/cellpose_residual_on_style_on_concatenation_off_train_2022_08_07_23_40_19.334298 --chan 0 --chan2 0 --diameter 0.0 --flow_threshold 0.4 --cellprob_threshold 0.0 --save_tif --no_npy --use_gpu --verbose]
INFO: Cellpose2D: creating new log file
INFO: Cellpose2D: 2022-08-08 10:04:14,114 [INFO] WRITING LOG OUTPUT TO C:\Users\xxx\.cellpose\run.log
INFO: Cellpose2D: 2022-08-08 10:04:14,285 [INFO] ** TORCH CUDA version installed and working. **
INFO: Cellpose2D: 2022-08-08 10:04:14,286 [INFO] >>>> using GPU
INFO: Cellpose2D: 2022-08-08 10:04:14,288 [INFO] >>>> running cellpose on 6 images using chan_to_seg GRAY and chan (opt) NONE
INFO: Cellpose2D: 2022-08-08 10:04:14,289 [INFO] >>>> loading model C:/ABBA/QuPath-0.3.2/My/location/cellpose_residual_on_style_on_concatenation_off_train_2022_08_07_23_40_19.334298
INFO: Cellpose2D: 2022-08-08 10:04:16,124 [INFO] >>>> model diam_mean =  30.000 (ROIs rescaled to this size during training)
INFO: Cellpose2D: 2022-08-08 10:04:16,124 [INFO] >>>> model diam_labels =  24.321 (mean diameter of training ROIs)
INFO: Cellpose2D: 2022-08-08 10:04:16,125 [INFO] >>>> not using cyto, cyto2, or nuclei model, cannot auto-estimate diameter
INFO: Cellpose2D: 2022-08-08 10:04:16,125 [INFO] >>>> using diameter 24.321 for all images
INFO: Cellpose2D: 2022-08-08 10:04:16,128 [INFO] 0%|          | 0/6 [00:00<?, ?it/s]
INFO: Cellpose2D: 2022-08-08 10:04:20,661 [INFO] 17%|#6        | 1/6 [00:04<00:22,  4.53s/it]
INFO: Cellpose2D: 2022-08-08 10:04:21,946 [INFO] 33%|###3      | 2/6 [00:05<00:10,  2.62s/it]
INFO: Cellpose2D: 2022-08-08 10:04:22,881 [INFO] 50%|#####     | 3/6 [00:06<00:05,  1.85s/it]
INFO: Cellpose2D: 2022-08-08 10:04:23,438 [INFO] No cell pixels found.
INFO: Cellpose2D: 2022-08-08 10:04:23,521 [INFO] 67%|######6   | 4/6 [00:07<00:02,  1.37s/it]
INFO: Cellpose2D: 2022-08-08 10:04:24,739 [INFO] 83%|########3 | 5/6 [00:08<00:01,  1.32s/it]
INFO: Cellpose2D: 2022-08-08 10:04:25,625 [INFO] 100%|##########| 6/6 [00:09<00:00,  1.17s/it]
INFO: Cellpose2D: 2022-08-08 10:04:25,625 [INFO] 100%|##########| 6/6 [00:09<00:00,  1.58s/it]
INFO: Cellpose2D: 2022-08-08 10:04:25,625 [INFO] >>>> completed in 11.340 sec
INFO: Virtual Environment Runner Finished
INFO: Cellpose command finished running
INFO: Getting objects for C:\HistologyData\1085426\cellposetraining\cellpose-temp\Temp_16954_9672_cp_masks.tif
INFO: Getting objects for C:\HistologyData\1085426\cellposetraining\cellpose-temp\Temp_16954_8964_cp_masks.tif
INFO: Getting objects for C:\HistologyData\1085426\cellposetraining\cellpose-temp\Temp_17744_8964_cp_masks.tif
INFO: Getting objects for C:\HistologyData\1085426\cellposetraining\cellpose-temp\Temp_16954_10696_cp_masks.tif
INFO: Getting objects for C:\HistologyData\1085426\cellposetraining\cellpose-temp\Temp_17744_10696_cp_masks.tif
INFO: Getting objects for C:\HistologyData\1085426\cellposetraining\cellpose-temp\Temp_17744_9672_cp_masks.tif
INFO: Getting objects for C:\HistologyData\1085426\cellposetraining\cellpose-temp\Temp_17744_10696_cp_masks.tif Done
INFO: Getting objects for C:\HistologyData\1085426\cellposetraining\cellpose-temp\Temp_17744_8964_cp_masks.tif Done
INFO: Getting objects for C:\HistologyData\1085426\cellposetraining\cellpose-temp\Temp_16954_8964_cp_masks.tif Done
INFO: Getting objects for C:\HistologyData\1085426\cellposetraining\cellpose-temp\Temp_16954_10696_cp_masks.tif Done
INFO: Getting objects for C:\HistologyData\1085426\cellposetraining\cellpose-temp\Temp_17744_9672_cp_masks.tif Done
INFO: Getting objects for C:\HistologyData\1085426\cellposetraining\cellpose-temp\Temp_16954_9672_cp_masks.tif Done
INFO: Making measurements for Annotation (Rectangle) (22 objects)
INFO: Done!

Based on the log, I felt that 22 objects are detected but are not annoated on my images somehow? (I could be wrong). I also checked those tiles and I do see masks on them.  
![image](https://user-images.githubusercontent.com/62570054/183390363-28c43b56-3d87-4d7a-b981-b94f5cf8a2a3.png)
Do you have any explanation? 
**Below is my script**
def imageData = getCurrentImageData()
def stored_annotations = getAnnotationObjects()
if (stored_annotations.isEmpty()) {
    Dialogs.showErrorMessage("Cellpose", "Please select a parent object!")
    return
}
import qupath.ext.biop.cellpose.Cellpose2D

def pathModel = "C:/ABBA/QuPath-0.3.2/My/location/cellpose_residual_on_style_on_concatenation_off_train_2022_08_07_23_40_19.334298"

def cellpose = Cellpose2D.builder( pathModel)
        .pixelSize( 0.5 )              // Resolution for detection
        .channels( 'tdTom')            // Select detection channel(s)C
        //.cellExpansion(5.0)            // Approximate cells based upon nucleus expansion
        .measureShape()                // Add shape measurements
        .measureIntensity()            // Add cell measurements (in all compartments)  
        .useGPU()                      // Optional: Use the GPU if configured, defaults to CPU only
        .createAnnotations()
        .build()

cellpose.detectObjects(imageData, stored_annotations)
println 'Done!'

**Comment by lacan** [comment]
Can you please post this usage issue to the https://forum.image.sc
that way we can debug it with the community and help others who might be having the same problems.
